internal/manifests/manifests.go

```diff
@@ -202,16 +282,30 @@ func (m *Manifests) UpdateClusterManifestInternal(ctx context.Context, params op
 		return nil, err
 	}
 
+	isCustomManifest := m.isCustomManifest(ctx, params.ClusterID.String(), srcFolder, srcFileName)
```

Comment:
And even if a user updates a generated manifest, does that not make it a user manifest?

Reply:
I don't think there is really an opportunity to do this as manifest generation is one of the last actions before install starts, as part of the preparation phase (I think.)
This would present no opportunity for the user to override this.